operator/install/kube/wasme/Chart.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+home: https://docs.solo.io/web-assembly-hub/latest
+icon: https://raw.githubusercontent.com/solo-io/wasme/master/docs/content/img/logo.png
+name: Wasme Operator
+sources:
+- https://github.com/solo-io/wasme
+version: v0.0.1
```

Comment:
In our other repos we have the directory structure `install/helm/<project>` Can we stick to the same here?

Reply:
the difference here is that wasme is not an operator-only project. `install` is associated only with the Operator, not the CLI version of wasme

Reply:
That is fine, I meant `operator/install/helm/<project>`